scripts/index.js

```diff
@@ -1 +1,149 @@
-console.log(`There are ${characters.length} characters in the array.`);
\ No newline at end of file
+// console.log(`There are ${characters.length} characters in the array.`);
+
+// // How many characters names start with "A"?
+
+// function startWithA(characters) {
+//     let nameA = [];
+//     let count = 0;
+//     characters.forEach(function (array) {
+//         if (array.name.startsWith("A")) {
+//             nameA.push(array.name);
+//             count += 1;
+//         }
+//     });
+//     // console.log(nameA);
+//     console.log(count);
+// }
+
+// startWithA(characters);
+
+// // How many characters names start with "Z"?
+
+// function startWithZ(characters) {
+//     let nameZ = [];
+//     let count = 0;
+//     characters.forEach(function (array) {
+//         if (array.name.startsWith("Z")) {
+//             nameZ.push(array.name);
+//             count += 1;
+//         }
+//     });
+//     // console.log(nameZ);
+//     console.log(count);
+// }
+
+// startWithZ(characters);
+
+// // How many characters are dead?
+
+function dead(characters) {
+    let nameDead = [];
+    let count = 0;
+    characters.forEach(function (array) {
+        if (array.died === "") {
+            nameDead.push(array.name);
+            count += 1;
+        }
+    });
+    console.log(nameDead);
+    console.log(count);
+}
+
+dead(characters);
+
+// // Who has the most titles?
+
+// function mostTitle(characters) {
+//     let count = 0;
+//     let name = [];
+//     let nameWithSameNumbers = [];
+//     characters.forEach(function (array) {
+//         if (array.titles.length > count) {
+//             count = array.titles.length;
+//             name = array.name;
+//         }
+//     });
+//     characters.forEach(function (array) {
```

Comment:
since this and line 60 are identical, it is a clue that the code can be condensed. any time you notice yourself repeating something, see if you can refactor them:
`function mostTitle(characters) {`
 ` let count = 0;`
 ` characters.forEach(function(array) {`
  `  if (array.titles.length > count) {`
      `count = array.titles.length;`
     ` name = array.name;`
  `  }`
 ` });`
  `console.log(`
  `  ``Who has the most titles?: ${name} \n Number of titles: ${count}`` `
`  );`
`};`
^^ This was your code, I just took it and tinkered for a minute to see what I could cut out! 🙂